install_config/web_console_customization.adoc

```diff
@@ -537,20 +537,6 @@ how to configure HAProxy routers to allow wildcard routes].
 Sometimes features are available in Technology Preview. By default, these
 features are disabled in the web console and hidden from end users.
 
-Web console features currently in Technology Preview include:
```

Comment:
... "that are hidden in the console", we may have things in tech preview that we don't need to hide in the console